src/plugins/vis_type_timelion/server/series_functions/es/index.js

```diff
@@ -129,9 +129,15 @@ export default new Datasource('es', {
 
     const body = buildRequest(config, tlConfig, scriptedFields, esShardTimeout);
 
-    const { callAsCurrentUser: callWithRequest } = tlConfig.esDataClient();
-    const resp = await callWithRequest('search', body);
-    if (!resp._shards.total) {
+    const resp = await tlConfig.esDataClient(
+      tlConfig.context,
+      {
```

Comment:
Let's move` { params: body }` into `buildRequest`.  We execute that method only from one place

Reply:
done